server/odc-service/src/main/java/com/oceanbase/odc/service/session/ConnectSessionService.java

```diff
@@ -382,6 +389,37 @@ public ConnectionSession nullSafeGet(@NotNull String sessionId, boolean autoCrea
         return session;
     }
 
+    public ConnectionSession nullSafeGetSkipPermissionCheck(@NotNull String sessionId,
```

Comment:
duplicated codes, can we reuse these codes with the method
`public ConnectionSession nullSafeGet(@NotNull String sessionId, boolean autoCreate)`?

Reply:
okay, i'll try to reuse this codes